style.css

```diff
@@ -0,0 +1,216 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+body {
+  font-family: 'Poppins';
+}
+
+.topheader {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+  }
+  
+nav {
+    width: 600px;
+  }
+  
+nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+nav a {
+    text-decoration: none;
+    color: black;
+  }
+  
+.blackHeart {
+    width: 20px;
+    margin-right: 1rem;
+  }
+
+.bottomheader {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+  }
+
+  .npmlogo {
+      width: 120px;
+      margin: 16px 32px 0px 0px;
+      padding: 10px;
+      flex: 1;
+  }
+
+.searchbar{
+    display: flex;
+    justify-content: right;
+    margin: 10px;
+}
+
+.searchbar input {
+    background-color: rgba(0, 0, 0, .05);
+    padding: 16px 32px;
+    border: none;
+    width: 600px;
+}
+
+.searchbar button {
+    background-color: #fb3e44;
+    color: white;
+    padding: 8px 32px;
+    border-color: #fb3e44;
+    border: none;
+    flex: 1;
+}
+
+.link-signup-button {
+    border-style: solid;
+    border-color: rgba(0, 0, 0, .05);
+    color: black;
+    background-color: white;
+    padding: 9px;
+    text-decoration: none;
+}
+
+.link-login-button {
+    color: black;
+    background-color: white;
+    padding: 9px;
+    text-decoration: none;
+}
+
+#section1 {
```

Comment:
Typically, id/# are used for navigation purposes, it is always better to use classes over id or tag names